src/containers/StudyCreateForm/StudyCreateForm.tsx

```diff
@@ -359,7 +411,13 @@ function StudyCreateFormContainer() {
                   onChange={handleChange}
                 />
               </StudyDescriptionWrapper>
-              <Button type='submit' disabled={isSubmitting}>
+              <Button
+                type='submit'
+                disabled={isSubmitting}
+                onClick={() => {
+                  handleClick(values);
+                }}
```

Comment:
p3;
다음과 같이 해도 잘작동하지 않을 까요?
개인적인 style이면 Pass 해주세요 :)

```suggestion
                onClick={handleClick(values)}
```

Reply:
팽님이 suggestion 해주신 대로 하면 ts error가 발생해서 이렇게 작성했습니다.😥